src/webgpu/shader/validation/decl/var_access_mode.spec.ts

```diff
@@ -130,7 +137,7 @@ g.test('explicit_access_mode')
         .combine('addressSpace', kNonHandleAddressSpaces)
         .expand('info', infoExpander)
         .combine('explicitSpace', [true, false])
-        .filter(t => compatibleAS(t))
+        .filter(t => varDeclCompatibleAddressSpace(t))
```

Comment:
I'm not convinced that `varDeclCompatibleAddressSpace` is needed, as it's a trivial expression and forces me jump around looking for the definition. Suggest inlining:

```suggestion
        .filter(t => t.info.spell !== 'must' || t.explicitSpace) // Remove decls requiring an address space that don't have one provided
```